app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
By switchAdditionalVisibility doesn't get called at all, do you mean like if you add an alert statement right after `switchAdditionalVisibility: function() {`, line 14, it won't be called at all?